2020-11-25 04:33:32,082: DEBUG: NEW RUN: Client, TEST: Write_Multiple_Clients_Two_Threads2
2020-11-25 04:33:32,082: DEBUG: The client is starting.
2020-11-25 04:33:32,082: DEBUG: The client is starting on port 52624
2020-11-25 04:33:32,083: DEBUG: Sent WRQ: b'\x00\x02shutdown.txt\x00netascii\x00'
2020-11-25 04:33:32,085: DEBUG: File exists, can send it
2020-11-25 04:33:32,085: DEBUG: First packet received. Raw bytes: b'\x00\x05\x00\x02filename already exists at destination\x00'
2020-11-25 04:33:32,085: DEBUG: Sending: shutdown.txt
2020-11-25 04:33:32,086: DEBUG: New packet from: ('127.0.0.1', 54605), contents: b'\x00\x05\x00\x02filename already exists at destination\x00'
2020-11-25 04:33:32,106: ERROR: Shutting down due to error packet. Error code: 2, error message: b'filename already exists at destination'
2020-11-25 04:33:32,106: WARNING: File didnt close on its own, closing now
2020-11-25 04:33:32,186: DEBUG: Client finished.
2020-11-25 04:33:32,186: DEBUG: Closing Socket
